package.json

```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "node-sass": "^4.11.0",
```

Comment:
Generally, we're avoiding to allow auto-upgrade for packages. Let's remove `^` sign at the beginning of the package version for all packages.